examples/quickstart.sh

```diff
@@ -0,0 +1,8 @@
+# Train PPO agent on cartpole and collect expert demonstrations. Tensorboard logs saved in `quickstart/rl/`
```

Comment:
Do we need this file? It's not referenced anywhere, and seems to duplicate the README. There is something nice about being able to actually *run* it, but not sure it's worth duplication. (Sadly no include directive in Markdown.)

Reply:
Ah I see you added a test to check the snipped hasn't diverged. That removes my concern re: code duplication. But we should probably still mention the files existence from the README?